kfs-core/src/main/resources/edu/arizona/kfs/fp/businessobject/datadictionary/ProcurementCardHolder.xml

```diff
@@ -205,6 +205,9 @@
 		<property name="label" value="Card Holder Postal (ZIP) Code" />
 		<property name="shortLabel" value="Code" />
 		<property name="required" value="false"/>
+		<property name="validationPattern">
+			<bean parent="AnyCharacterValidationPattern"/>
+		</property>

```

Comment:
Do we need to add the maxLenght for this field as I didn't see it in the parent xml:
<property name="maxLength" value="20" />